### PR TITLE
Adding Sao Goncalo - RJ

### PR DESCRIPTION
**Pull request by joaopclima**

### CODE REVIEW

**Comment by jaswdr** [comment]
@joaopclima can you review this PR? I tried it locally and got this error:
```
2021-07-12 18:14:31 [scrapy.spidermiddlewares.httperror] INFO: Ignoring response <404 https://servicos.pmsg.rj.gov.br/diario_oficial>: HTTP status code is not handled or not allowed
```
It seems that the URL is wrong.

**Comment by joaopclima** [comment]
Sure! I will review soon

File: data_collection/gazette/spiders/rj_sao_goncalo.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from gazette.spiders.base.instar import BaseInstarSpider
+
+
+class RjSaoGoncaloSpider(BaseInstarSpider):
+
+    TERRITORY_ID = "3304904"
+    name = "rj_sao_goncalo"
+    allowed_domains = ["www.saogoncalo.rj.gov.br"]
+    start_urls = ["https://servicos.pmsg.rj.gov.br/diario_oficial"]
```
Comment (anapaulagomes) [inline data_collection/gazette/spiders/rj_sao_goncalo.py:9]: This page doesn't exist but this one does https://servicos.pmsg.rj.gov.br/diario_oficial.php. But after running it with this one and with https://www.saogoncalo.rj.gov.br/diario-oficial/, none are working.
By looking at other portals like http://www.altoalegre.sp.gov.br/portal/diario-oficial that also implement `BaseInstarSpider`, it seems that São Gonçalo isn't using Instar. Could you check this, please? @joaopclima

**Review by anapaulagomes — CHANGES_REQUESTED** [review]
In case this isn't a portal that uses Instar, you will have to implement it for São Gonçalo's portal or we will have to close the PR. Count on us to help you with it. :)

**Comment by rennerocha** [comment]
Closed as stale.

**Pull request closed without merging**